modules/adagioAnalyticsAdapter.js

```diff
@@ -266,7 +284,12 @@ function handlerBidWon(event) {
     logError('Error on Adagio Analytics Adapter - handlerBidWon', error);
   }
 
-  cache.updateAuction(event.auctionId, event.adUnitCode, {
+  const adagioAuctionCacheId = (
+    (event.latestTargetedAuctionId && event.latestTargetedAuctionId != event.auctionId)
```

Comment:
```suggestion
    (event.latestTargetedAuctionId !== event.auctionId)
```

Reply:
With:
- `event.latestTargetedAuctionId: undefined`
- `event.auctionId: string:"1"`

`(undefined && undefined != string:"1") -> false -> return event.auctionId -> "1"` ✅ 
`(undefined !== string:"1") -> true -> return event.latestTargetedAuctionId -> undefined` :x:

With:
- `event.latestTargetedAuctionId: string:"2"`
- `event.auctionId: string:"1"`

`(string:"2" && string:"2" != string:"1") -> false -> return event.auctionId -> "1"` ✅ 
`(string:"2" !== string:"1") -> true -> return event.latestTargetedAuctionId -> "2"` ✅ 

Or we consider that `event.latestTargetedAuctionId` can't be undefined ?

Reply:
My bad, we must keep `&&`. But keep `!==` strict equal